exercicios/para-casa/dataset_alunas.py

```diff
@@ -59,4 +59,129 @@
         "Presença": [True, True, True, False, True],
         "Participação": 7.5
     }
-}
\ No newline at end of file
+}
+
+
+
+print ("-------------------------hello world!!---------------------------------")
+print ("-------------------Esse é o sistema reprograma-------------------------")
+print ("-----------------------seja bem vinde ;) ------------------------------")
+print ("----------Para ter acesso a lista de alunas (digite 1)-----------------")
+print ("----------------------Para inserir aluna nova (digite 2)--------------")
+print ("-------------------Para consultar presença (digite 3)-----------------")
+print ("----------------Para consultar notas  (digite 4)------------------------")
+print ("----------------Para calcular média (digite 5)------------------------")
+print ("-----------------Para sair do sistema (digite 6)----------------------")
+
+ 
+
+ 
+
+     
+    
+def inclusao_novas_alunas ():
+    while True:
+     nome = str(input("Insira o nome: "))
+     sobrenome = input("Insira o seu sobrenome: ")
+     turma = input(" Insira a sua turma: ")
+   
+     notas = []
+     for i in range (3):
+       nota = int(input (f"Insira as suas {i+1} notas: "))
+       notas.append(nota)
+       
+     presenca = []
+     for i in range (5):
+       aulas = bool(input(f"insira presença {i+1} com 'True' e falta com 'False' são 5 aulas: ")) # returnar, criar função bool
+       presenca.append(presenca)
+    
+     participacao = int(input("Insira sua nota de participação de 0 á 10: "))
+   
+     salvamento_dos_dados_aluna = (nome,sobrenome,turma,notas,presenca,participacao)
+
+     print (f"olá, {nome} seja bem vinde a rede reprograma! ")
+     break
+
+
+
+def salvamento_dos_dados_aluna (nome,sobrenome,turma,notas,presenca,participacao):
+    chave = (nome)
+    dataset = [chave] = { 
+        "Turma": turma,
+        "Notas": notas,
+        "Presença": presenca,
+        "Participação": participacao
+    }
+
+    print (f"a aluna {nome} foi adicionada com sucesso!!")
+    return qual_numero
+
+def consultar_dados_alunas ():
+ nome_aluna_cadastrada = input("insira o nome da aluna que deseja ver os dados :")
+
+ sobrenome_aluna_cadastrada = input("insira o sobrenome da aluna que deseja ver os dados :")
+
+
+ if (nome_aluna_cadastrada, sobrenome_aluna_cadastrada) not in dataset:
+    print ("esse nome não está no sistema!!")
+ 
+ else:
+    print (f"o seu nome  é {nome_aluna_cadastrada} e sobrenome {sobrenome_aluna_cadastrada}")
+    
+    return nome_aluna_cadastrada,sobrenome_aluna_cadastrada
+    
```

Comment:
Adorei que você integrou a parte de consulta em uma única função. Cuidado com excesso de espaços nesta função.
Neste caso, como estamos consultado a aluna, não precisamos retornar ao usuário quando a aluna foi encontrada, pois caso ela seja encontrada, apenas retornamos os dados que o usuário espera, como notas, presença, etc..

**Sugestão:**
```suggestion
def consultar_dados_alunas ():
    nome_aluna_cadastrada = input("insira o nome da aluna que deseja ver os dados :")
    sobrenome_aluna_cadastrada = input("insira o sobrenome da aluna que deseja ver os dados :")
    
    if (nome_aluna_cadastrada, sobrenome_aluna_cadastrada) not in dataset:
        print ("Esse nome não está no sistema!")
        
    return nome_aluna_cadastrada,sobrenome_aluna_cadastrada   
```